t/sql/initial.sql

```diff
@@ -785,95 +785,95 @@ INSERT INTO orderable_link_type VALUES (742, 2);
+INSERT INTO release_status VALUES (3, 'Bootleg', NULL, 3, 'An unofficial/underground release that was not sanctioned by the artist and/or the record company. This includes unofficial live recordings and pirated releases.', '1156806e-d06a-38bd-83f0-cf2284a808b9');
+INSERT INTO release_status VALUES (4, 'Pseudo-Release', NULL, 4, 'An alternate version of a release where the titles have been changed. These don''t correspond to any real release and should be linked to the original release using the transl(iter)ation relationship.', '41121bb9-3413-3818-8a9a-9742318349aa');
+
+INSERT INTO series_alias_type VALUES (1, 'Series name', NULL, 0, NULL, '0c615dbc-c7d6-39b3-b8da-bef465ce3046');
+INSERT INTO series_alias_type VALUES (2, 'Search hint', NULL, 0, NULL, '8950366b-5ea3-32f2-bf74-ee482474c18b');
+
+INSERT INTO series_ordering_type VALUES (1, 'Automatic', NULL, 0, 'Sorts the items in the series automatically by their number attributes, using a natural sort order.', 'ec5fa247-c37a-39b3-b31d-bdac6207344e');
+INSERT INTO series_ordering_type VALUES (2, 'Manual', NULL, 1, 'Allows for manually setting the position of each item in the series.', '2950ba43-3532-39e9-a7d1-7dc7e271fa25');
+
+INSERT INTO series_type VALUES (1, 'Release group', 'release_group', NULL, 0, 'A series of release groups.', '4c1c4949-7b6c-3a2d-9d54-a50a27e4fa77');
+INSERT INTO series_type VALUES (2, 'Release', 'release', NULL, 1, 'A series of releases.', '52b90f1e-ff62-3bd0-b254-5d91ced5d757');
+INSERT INTO series_type VALUES (3, 'Recording', 'recording', NULL, 2, 'A series of recordings.', 'dd968243-7128-30a2-81f0-79843430a8e2');
+INSERT INTO series_type VALUES (4, 'Work', 'work', NULL, 3, 'A series of works.', 'b689f694-6305-3d78-954d-df6759a1877b');
+INSERT INTO series_type VALUES (5, 'Catalogue', 'work', 4, 0, 'A series of works which form a catalogue of classical compositions.', '49482ff0-fc9e-3b8c-a2d0-30e84d9df002');
+INSERT INTO series_type VALUES (6, 'Event', 'event', NULL, 5, 'A series of events.', '64640a2a-1c31-394d-b7b0-683a32ff9aff');
+INSERT INTO series_type VALUES (7, 'Tour', 'event', 6, 0, 'A series of related concerts by an artist in different locations.', '8ff6df0e-3dce-3bdf-bd57-d386c51b0060');
+INSERT INTO series_type VALUES (8, 'Festival', 'event', 6, 1, 'A recurring festival, usually happening annually in the same location.', '44e9f6b1-34a7-3a17-a5d5-07bb100d8887');
+INSERT INTO series_type VALUES (9, 'Run', 'event', 6, 2, 'A series of performances of the same show at the same venue.', '975e9447-dfa8-3e29-82d6-07908fe36f3e');
+
+INSERT INTO work_alias_type VALUES (1, 'Work name', NULL, 0, NULL, 'a18cab3f-0ae2-3978-8f75-dd9c09702b25');
+INSERT INTO work_alias_type VALUES (2, 'Search hint', NULL, 0, NULL, '02238bc1-dfd8-39a8-bbf8-c697747291ec');
+
+INSERT INTO work_type VALUES (29, 'Musical', NULL, 2, 'Musical theatre is a form of theatrical performance that combines songs, spoken dialogue, acting, and dance.', '9ca5e067-acf7-3cd6-baa4-92bf1975bf24');
```

Comment:
Can we keep each group sorted so it's easy to see what was actually added? (F5 in sublime, but you prob. knew that.)